src/test/compile-fail/rfc-2008-non-exhaustive/enum_field_attempt_create.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// FIXME issue #44109, PR #45394
+// gate-test-non_exhaustive
```

Comment:
This marker is used for feature gate tests, but this test is not a feature gate test.

Also, could you actually add a feature gate test? (Making sure a feature error is reported in `#[non_exhaustive]` is used without `feature(non_exhaustive)`).

Reply:
Same with other tests.

Reply:
I've added a proper feature gate test.